includes/class-wp-job-manager-usage-tracking-data.php

```diff
@@ -54,6 +56,33 @@ private static function get_employer_count() {
 		return $employer_query->total_users;
 	}
 
+	/**
+	 * Get the number of job categories that have a description.
+	 *
+	 * @since 1.30.0
+	 *
+	 * @return int Number of job categories with a description.
+	 **/
+	private static function get_job_category_has_description_count() {
+		$count = 0;
+		$terms = get_terms(
```

Comment:
TIL, while investigating #825, that the new signature for `get_terms()` (in the way you used it here) is annoyingly not backward compatible for WordPress instances before version 4.5. I think it will fatal ([#](https://github.com/WordPress/WordPress/blob/4.4.14/wp-includes/taxonomy.php#L1070)).

I punted on #825 for now. It is only half-way deprecated (doesn't throw notice) in WP core to use `get_terms( $taxonomy, $args );`, but still feels weird to use deprecated behavior on new code. We could do a version check here or we could create a global wrapper function (that does the version check).

Reply:
Bump minimum version? :D 4.1 is from 2014. WooCommerce minimum is 4.7.

Reply:
@spencerfinnell I think we should, but it might be nice to have some usage data for how many folks are pre-4.5 :).

Reply:
I think it is. But waiting a bit after some data makes sense. I would imagine it matches WC pretty well.

Reply:
@jom Do you mean do a WP version check and based on that, determine which version of `get_terms` to call? Or something else?

Reply:
Yes, a WP version check.

Reply:
But if I'm going to be calling an older version of `get_terms` anyway based on the version check, why not just forego a version check and call it using the older syntax regardless of version?

Reply:
That's fine because it doesn't throw any deprecation notice right now. It is also what we're doing elsewhere in the plugin (from #825).

Reply:
Its just a gamble if core with truly deprecate the first string argument or not. Unlikely they will but it does introduce the possibility of needing future updates.

Reply:
Fixed in https://github.com/Automattic/WP-Job-Manager/pull/1340/commits/1cc5ba30298a4ab7ebcd9d2909cee5f91d26d61f.